frontend/src/scenes/insights/filters/ActionFilter/ActionFilter.tsx

```diff
@@ -242,23 +242,21 @@ export const ActionFilter = React.forwardRef<HTMLDivElement, ActionFilterProps>(
             ) : null}
             {!singleFilter && (
                 <div className="ActionFilter-footer">
-                    {!singleFilter && (
```

Comment:
fly-by change: the `!singleFilter` check is already done a couple of lines above at the wrapping div. As such the check isn't needed again here.